emails/library/sendgrid.py

```diff
@@ -26,4 +26,4 @@ def send(self,
                 template_id=template_id,
             )
         except Exception as error:
-            return HttpResponseServerError(f'Could not send registration confirmation email: {str(error)}')
+            return HttpResponseServerError(f'Could not send email, emailable_type - {emailable_class_name}: {str(error)}')
```

Comment:
Might be useful to say “Could not send email for {emailable_class_name} - {emailable_id}: {str(error)}”, so that we know exactly which record to look at when an issue occurs.